pages/docs/tooling/tooling-aste.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Artificial Solver Testing Environment (ASTE)
+permalink: tooling-aste.html
+keywords: tooling, aste
+summary: "A guide to usage of artifical solver testing environment (ASTE) tool"
+---
+
+## Motivation
+
+ASTE is a tool which can be used to understand the dynamics of preCICE. It is a lightweight wrapper around preCICE which allows to investigate different mapping setups in terms of both runtime and accuracy. It can be replace individual participant for development purpouses. It consist of C++ and Python modules.
+
+## Installation
+
+ASTE relies on following requirements
+
+### C++ modules
+
+#### Required dependencies for C++
+
+- C++ compiler (with support for C++14, e.g. GCC version >= 5)
+- preCICE
+- MPI
+- CMake (version >= 3.1)
+- Boost (version >= 1.65.1)
+- vtk (Visualization ToolKit)
```

Comment:
I built vtk from source. Is there any better on Ubuntu 20.04?
Could we specify a minimal vtk version?

Reply:
Building VTK from source would be pain. 

https://packages.ubuntu.com/search?keywords=vtk
`sudo apt install libvtk7-dev` or preferable `sudo apt install libvtk9-dev` will work.

Specifying a minimal version is a hard choice, this also depends on the VTK files. The latest version can be recommended but not a minimum requirement.

Reply:
> Specifying a minimal version is a hard choice, this also depends on the VTK files. The latest version can be recommended but not a minimum requirement.

Let's add such a statement.

and could we maybe add a brief "For example, on Ubuntu 20.04, you get all dependencies with `sudo apt install ...`"?